src/streams/EditStreamCard.js

```diff
@@ -47,8 +53,12 @@ export default class EditStreamCard extends PureComponent<Props, State> {
   };
 
   render() {
-    const { initialValues, isNewStream } = this.props;
-    const { name } = this.state;
+    const { initialValues, isNewStream, streams } = this.props;
+    const { name, description, isPrivate } = this.state;
+
+    const isInvalidStreamName =
```

Comment:
Often I prefer the positive variant `isValid` as I find it reads better and we don't have to do the double negative to achieve valid `isValid` vs `!isInvalid`.

Reply:
I changed it from negative to positive logic.